### DISCUSSION TITLE
Add dotNET 8.0 support?

### DISCUSSION THREAD

**Discussion by TurnOffNOD** [gitextensions/gitextensions#11829] [General]
### Feature description

dotNET 6.0 will be out of support after November 12, 2024, so maybe gitextensions should add dotNET 8.0 support?

### Environment

- GitExtensions version:
 v4.2.1
- GIT version: 
  2.46.0
- OS version: 
  win 11 pro with lastest upgrade
- .NET version: 
  dotNET 6.0.32

**Comment by gerhardol** [comment]
A test build is available: https://github.com/gitextensions/gitextensions/discussions/11573
